udata/features/territories/models.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from flask import current_app, url_for
+
+__all__ = ('TerritoryDataset', 'ResourceBasedTerritoryDataset')
+
+
+class TerritoryDataset(object):
+    id = ''
+    title = ''
+    url_template = ''
+    description = ''
+
+    def __init__(self, territory):
+        self.territory = territory
+
+    @property
+    def url(self):
+        return self.url_template.format(code=self.territory.code)
+
+    @property
+    def slug(self):
+        return '{territory_id}-{id}'.format(
+            territory_id=self.territory.id.replace('/', '-'), id=self.id)
+
+    @property
+    def server(self):
+        return '{scheme}://{domain}'.format(
+            scheme=current_app.config['USE_SSL'] and 'https' or 'http',
+            domain=current_app.config['SERVER_NAME'])
+
+    @property
+    def api_url(self):
+        return '{server}/api/1/territories/{path}/'.format(
```

Comment:
There is no dedicated endpoint for that URL, maybe I should add one just to have the `url_for`?